autorel/changelog_generator/parser_config.py

```diff
@@ -0,0 +1,55 @@
+## Register your parsing functions here
+
+import re
+
+
+def pull_id_parser(msg_body):
+    """
+        Parses pull request id from commit message
+        => msg : Merge pull request #1234 from username/branch
+        => pull_id : 1234
+    """
+    if msg_body.startswith('Merge pull'):
+        tokens = msg_body.split()
+        pull_id = tokens[3][1:]
```

Comment:
Please do at least a range check in these functions, you may never know what comes after the "startswith" pattern.